lib/datasource/local/settings_storage.dart

```diff
@@ -176,7 +181,7 @@ class _SettingsStorage {
       if (values.containsKey(_kPrivateKeyBackedUp)) {
         _privateKeyBackedUp = values[_kPrivateKeyBackedUp] == 'true';
       } else {
-        _privateKeyBackedUp = false;
```

Comment:
I don't understand this

Private key backed up has a very simple logic it is true when the private key has been backed up, and false when it's not backed up

Import key -> backed up = true
Created new key in the app -> backed up = false

This what it needs to do. So I don't think we need this _kPrivateKeyBackedUpDefult variable?!

Reply:
const defualts moved to implementations 👍